.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,101 @@
+name: 🧪 E2E Tests
+
+on:
+  push:
+    branches: [main, stevank/eng-594/e2e-tests-setup, dev]
```

Comment:
@skosijer Remember to remove this before merging the PR.

Reply:
Still keeping this change here during the PR review.
I want to make sure any updates to the PR run the action for this branch.